apps/web/src/pages/integrations/components/ProviderCard.cy.tsx

```diff
@@ -0,0 +1,140 @@
+import { TestWrapper } from '../../../testing';
+import { ProviderCard } from './ProviderCard';
+import { ChannelTypeEnum, CredentialsKeyEnum } from '@novu/shared';
+import { ColorSchemeProvider } from '@mantine/core';
+
+const providerCardTestId = 'integration-provider-card';
+
+const provider = {
+  providerId: 'providerItem.id',
+  integrationId: 'integration.id',
+  displayName: 'providerItem.displayName',
+  channel: ChannelTypeEnum.EMAIL,
+  credentials: [
+    {
+      key: CredentialsKeyEnum.ApiKey,
+      displayName: 'API Key',
+      type: 'string',
+      required: true,
+    },
+    {
+      key: CredentialsKeyEnum.SecretKey,
+      displayName: 'Secret key',
+      type: 'string',
+      required: true,
+    },
+  ],
+  docReference: 'providerItem.docReference',
+  comingSoon: false,
+  betaVersion: false,
+  active: false,
+  connected: false,
+  logoFileName: { light: 'examplelight.svg', dark: 'exampledark.svg' },
+};
+
+it('light theme component', () => {
+  const colorScheme = 'light';
+
+  cy.mount(
+    <TestWrapper>
+      <ColorSchemeProvider colorScheme={colorScheme} toggleColorScheme={() => {}}>
+        <ProviderCard provider={provider} onConnectClick={() => {}} />
+      </ColorSchemeProvider>
+    </TestWrapper>
+  );
+
+  cy.getByTestId(providerCardTestId)
+    .find('img')
+    .should('have.attr', 'src', `/static/images/providers/${colorScheme}/${provider.logoFileName[`${colorScheme}`]}`)
+    .should('have.attr', 'alt', provider.displayName);
+});
+
+it('dark theme component', () => {
```

Comment:
@shtelzerartem when writing tests we usually try to construct the message to be read in the following way:
```suggestion
it('should change the image based on the colorScheme selected by user', () => {
```

Reply:
I will change it to this, ok? 
For dark:
```javascript
it('should change the image based on the colorScheme (dark) selected by user', () => {
```
For light:
```javascript
it('should change the image based on the colorScheme (light) selected by user', () => {
```